src/bin/deku_node.ml

```diff
@@ -68,13 +68,27 @@ let handle_block_by_hash =
       Ok block)
 
 (* POST /block-level *)
-(* Retrieve height of the chain? *)
+(* Retrieve height of the chain *)
 let handle_block_level =
   handle_request
     (module Network.Block_level)
     (fun _update_state _request ->
       Ok { level = Flows.find_block_level (Server.get_state ()) })
 
+(* POST /block-by-level *)
+(* Retrieves the block at the given level if it exists *)
+let handle_block_by_level =
+  handle_request
+    (module Network.Block_by_level_spec)
+    (fun _update_state request ->
+      let state = Server.get_state () in
+      let block =
+        List.find_opt
+          (fun block ->
+            Int64.equal block.Protocol.Block.block_height request.level)
+          state.applied_blocks in
+      Ok block)
+
```

Comment:
I'm a bit annoyed by the fact that we use two terms, "level" and "height", to designate the same thing. Can we decide which one we want to use once and for all? Switching to `level` seems acceptable as it is the term used in Tezos, keeping `height` is OK as well because inertia, also the term seems to be more common outside of Tezos.

Reply:
This seems out of scope for this PR. I'm open to a PR that standardizes usage across the code base though.